kasparovsyncd/mqtt/transactions.go

```diff
@@ -20,26 +20,11 @@ const (
 	UnacceptedTransactionsTopic = "transactions/unaccepted"
 )
 
-// PublishTransactionsNotifications publishes notification for each transaction of the given block
-func PublishTransactionsNotifications(rawTransactions []rpcmodel.TxRawResult) error {
-	if !isConnected() {
-		return nil
-	}
-
-	transactionHashes := make([]string, len(rawTransactions))
-	for i, tx := range rawTransactions {
-		transactionHashes[i] = tx.Hash
-	}
-
-	dbTransactions, err := dbaccess.TransactionsByHashes(dbaccess.NoTx(), transactionHashes,
-		dbmodels.TransactionRecommendedPreloadedFields...)
-	if err != nil {
-		return err
-	}
-
+// publishTransactionsNotifications publishes notifications for each transaction of the given transactions
+func publishTransactionsNotifications(dbTransactions []*dbmodels.Transaction) error {
 	for _, dbTransaction := range dbTransactions {
 		transaction := apimodels.ConvertTxModelToTxResponse(dbTransaction)
-		err = publishTransactionNotifications(transaction, TransactionsTopic)
+		err := publishTransactionNotifications(transaction, TransactionsTopic)
```

Comment:
publishTransactionsNotifications and publishTransactionNotifications are waaaaayyy to similar.
Please find rename or restructure in some way to create a better destinction.